docs/css/avatar.css

```diff
@@ -0,0 +1,42 @@
+.bat-avatar-xl {
+    height: 12rem;
+    width: 12rem;
+    border-radius: 50%;
+    overflow: hidden;
+}
+
+
+.bat-avatar-large {
```

Comment:
You can shorten **large** to **lg** to maintain consistency in the code.

Reply:
done